x-pack/plugins/maps/public/classes/sources/es_search_source/top_hits/wizard.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { CreateSourceEditor } from './create_source_editor';
+import { LayerWizard, RenderWizardArguments } from '../../../layers/layer_wizard_registry';
+import { VectorLayer } from '../../../layers/vector_layer';
+import { LAYER_WIZARD_CATEGORY } from '../../../../../common/constants';
+import { TopHitsLayerIcon } from '../../../layers/icons/top_hits_layer_icon';
+import { ESSearchSourceDescriptor } from '../../../../../common/descriptor_types';
+import { ESSearchSource } from '../es_search_source';
+
+export const esTopHitsLayerWizardConfig: LayerWizard = {
+  categories: [LAYER_WIZARD_CATEGORY.ELASTICSEARCH],
+  description: i18n.translate('xpack.maps.source.topHitsDescription', {
+    defaultMessage: 'Display the most relevant documents per entity',
```

Comment:
Maybe add an example: `e.g. to show the most recent document for a given entity`. (?)